frontend/src/core/models/enums/task-priority.ts

```diff
@@ -0,0 +1,5 @@
+export enum TaskPriority {
+    Low,
+    Medium,
+    High,
+}
```

Comment:
All this parts will be customizable, so, please don't use enums

Reply:
Ok, i will remove them from here and from db